runtime/common/src/constants.rs

```diff
@@ -66,18 +66,23 @@ pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
 
 // Unit = the base number of indivisible units for balances
-pub const UNIT: Balance = 100_000_000;
-pub const MILLIUNIT: Balance = 1_000_000_000;
-pub const MICROUNIT: Balance = 1_000_000;
-
-/// The existential deposit. Set to 1/10 of the Connected Relay Chain.
-pub const EXISTENTIAL_DEPOSIT: Balance = MILLIUNIT;
-
-/// Generates an balance based on amount of items and bytes
-/// Items are each worth 20 Tokens
-/// Bytes each cost 1/1_000 of a Token
-pub const fn deposit(items: u32, bytes: u32) -> Balance {
-	items as Balance * 20 * UNIT + (bytes as Balance) * UNIT / 1_000
+pub mod currency {
+	use common_primitives::node::Balance;
+
+	/// The existential deposit. Set to 1/10 of the Connected Relay Chain.
+	pub const EXISTENTIAL_DEPOSIT: Balance = 100 * CENTS;
+
+	pub const UNITS: Balance = 100_000_000;
```

Comment:
Is it possible to base this off of a constant that produces the DECIMAL places? That way the everything ties back to decimal places?